### PR TITLE
Add Cisco Aironet wireless access points

### PR DESCRIPTION
**Pull request by abu-rehan**
This pull request adds a number of cisco access points. They are from different series and so I have simply named the branch as aironet-access-points. This is my first contribution and so, I may be off on something. Please let me know of any edits I have to make.

### CODE REVIEW

File: device-types/Cisco/AIR-AP1121G-E-K9.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 1121G-E
+slug: cisco-air-ap1121g-e-k9
+part_number: AIR-AP1121G-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP1121G-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-AP1121G-E-K9.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    type: 100base-tx
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP1121G-E-K9.yaml:12]: Should be "Dot11Radio0"

File: device-types/Cisco/AIR-LAP1242N-E-K9.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+slug: cisco-air-lap1242n-e-k9
+part_number: AIR-LAP1242N-E-K9
+u_height: 0
+is_full_depth: false
+console-ports:
+  - name: con0
+    type: rj-45
+    mgmt_only: true
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-LAP1242N-E-K9.yaml:13]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-LAP1242N-C-K9.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+slug: cisco-air-lap1242n-c-k9
+part_number: AIR-LAP1242N-C-K9
+u_height: 0
+is_full_depth: false
+console-ports:
+  - name: con0
+    type: rj-45
+    mgmt_only: true
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-LAP1242N-C-K9.yaml:13]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-LAP1242N-A-K9.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+slug: cisco-air-lap1242n-a-k9
+part_number: AIR-LAP1242N-A-K9
+u_height: 0
+is_full_depth: false
+console-ports:
+  - name: con0
+    type: rj-45
+    mgmt_only: true
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-LAP1242N-A-K9.yaml:13]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP3702I-Z-K9.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3702I-Z
+slug: cisco-air-cap3702i-z-k9
+part_number: AIR-CAP3702I-Z-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3702I-Z-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP3702E-E-K9.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3702E-E
+slug: cisco-air-cap3702e-e-k9
+part_number: AIR-CAP3702E-e-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3702E-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP3602I-T-K9.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3602I-T
+slug: cisco-air-cap3602i-t-k9
+part_number: AIR-CAP3602I-T-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602I-T-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP3602I-S-K9.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3602I-S
+slug: cisco-air-cap3602i-s-k9
+part_number: AIR-CAP3602I-S-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602I-S-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP3602I-K-K9.yaml [hunk 10]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3602I-K
+slug: cisco-air-cap3602i-k-k9
+part_number: AIR-CAP3602I-K-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602I-K-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP3602I-C-K9.yaml [hunk 11]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3602I-C
+slug: cisco-air-cap3602i-c-k9
+part_number: AIR-CAP3602I-C-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602I-C-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP3602E-E-K9.yaml [hunk 12]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3602E-E
+slug: cisco-air-cap3602e-e-k9
+part_number: AIR-CAP3602E-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602E-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP3502I-E-K9.yaml [hunk 13]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3502I-e
+slug: cisco-air-cap3502i-e-k9
+part_number: AIR-CAP3502I-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3502I-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP3502E-E-K9.yaml [hunk 14]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3502E-E
+slug: cisco-air-cap3502e-e-k9
+part_number: AIR-CAP3502E-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3502E-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP2602I-E-K9.yaml [hunk 15]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 2602I-E
+slug: cisco-air-cap2602i-e-k9
+part_number: AIR-CAP2602I-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP2602I-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP1552SA-E-K9.yaml [hunk 16]
```diff
@@ -0,0 +1,16 @@
...
+slug: cisco-air-cap1552sa-e-k9
+part_number: AIR-CAP1552SA-E-K9
+u_height: 0
+is_full_depth: false
+power-ports:
+  - name: PS1
+    type: other
+    maximum_draw: 47
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP1552SA-E-K9.yaml:13]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP1552H-E-K9.yaml [hunk 17]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 1552H-E
+slug: cisco-air-cap1552h-e-k9
+part_number: AIR-CAP1552H-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP1552H-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-AP3802I-Z-K9.yaml [hunk 18]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3802I-Z
+slug: cisco-air-ap3802i-z-k9
+part_number: AIR-AP3802I-Z-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP3802I-Z-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-AP3802I-T-K9.yaml [hunk 19]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3802I-T
+slug: cisco-air-ap3802i-t-k9
+part_number: AIR-AP3802I-T-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP3802I-T-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-AP3802I-E-K9.yaml [hunk 20]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3802I-E
+slug: cisco-air-ap3802i-e-k9
+part_number: AIR-AP3802I-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP3802I-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-AP3802E-E-K9.yaml [hunk 21]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 3802E-E
+slug: cisco-air-ap3802e-e-k9
+part_number: AIR-AP3802E-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP3802E-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-AP1562E-B-K9.yaml [hunk 22]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 1562E-B
+slug: cisco-air-ap1562e-b-k9
+part_number: AIR-AP1562E-B-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP1562E-B-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-AP1562E-B-K9.yaml [hunk 23]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP1562E-B-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-AP3802E-E-K9.yaml [hunk 24]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP3802E-E-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-AP3802I-E-K9.yaml [hunk 25]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP3802I-E-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-AP3802I-T-K9.yaml [hunk 26]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP3802I-T-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-AP3802I-Z-K9.yaml [hunk 27]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP3802I-Z-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP1552H-E-K9.yaml [hunk 28]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP1552H-E-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP1552SA-E-K9.yaml [hunk 29]
```diff
@@ -0,0 +1,16 @@
...
+    type: 1000base-t
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP1552SA-E-K9.yaml:15]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP2602E-E-K9.yaml [hunk 30]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 2602E-E
+slug: cisco-air-cap2602e-e-k9
+part_number: AIR-CAP2602E-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: Port 1
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP2602E-E-K9.yaml:9]: Physically this is "Ethernet", Logically, it is generally "GigabitEthernet0"

File: device-types/Cisco/AIR-CAP2602E-E-K9.yaml [hunk 31]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP2602E-E-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP2602I-E-K9.yaml [hunk 32]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP2602I-E-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3502E-E-K9.yaml [hunk 33]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3502E-E-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3502I-E-K9.yaml [hunk 34]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3502I-E-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3602E-E-K9.yaml [hunk 35]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602E-E-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3602I-C-K9.yaml [hunk 36]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602I-C-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3602I-K-K9.yaml [hunk 37]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602I-K-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3602I-S-K9.yaml [hunk 38]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602I-S-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3602I-T-K9.yaml [hunk 39]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3602I-T-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3702E-E-K9.yaml [hunk 40]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3702E-E-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3702I-Z-K9.yaml [hunk 41]
```diff
@@ -0,0 +1,13 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3702I-Z-K9.yaml:12]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-LAP1242N-A-K9.yaml [hunk 42]
```diff
@@ -0,0 +1,17 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-LAP1242N-A-K9.yaml:16]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-LAP1242N-C-K9.yaml [hunk 43]
```diff
@@ -0,0 +1,17 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-LAP1242N-C-K9.yaml:16]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-LAP1242N-E-K9.yaml [hunk 44]
```diff
@@ -0,0 +1,17 @@
...
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-LAP1242N-E-K9.yaml:16]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: device-types/Cisco/AIR-AP1121G-E-K9.yaml [hunk 45]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 1121G-E
+slug: cisco-air-ap1121g-e-k9
+part_number: AIR-AP1121G-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
+    type: 100base-tx
```
Comment (abu-rehan) [inline device-types/Cisco/AIR-AP1121G-E-K9.yaml:10]: @DanSheps Do you mean this?
```suggestion
    type: ethernet
```

**Pull request closed without merging**